src/test/java/dk/kb/present/TestUtil.java

```diff
@@ -64,15 +69,37 @@ public static String getTransformedFromConfigWithAccessFields(YAML config, Strin
 		String mods = Resolver.resolveUTF8String(xmlResource);
 		HashMap<String, String> accessFields = XsltCopyrightMapper.applyXsltCopyrightTransformer(mods);
         accessFields.put("recordID", "ds.test:" + Path.of(xmlResource).getFileName().toString());
+		accessFields.put("origin", "ds.test");
 		//System.out.println("access fields:"+accessFields);
 		return transformer.apply(mods, accessFields);
 	}
 
 	/**
 	 * Transform the input MODS record with an XSLT and return as pretty JSON
 	 */
-	public static void prettyPrintSolrJsonFromMetadata(String xslt, String record) throws Exception {
-		String solrString = getTransformedWithAccessFieldsAdded(xslt, record);
+	public static void prettyPrintSolrJsonFromMods(String record) throws Exception {
+		String yamlStr =
+				"stylesheet: '" + MODS2SOLR + "'\n" +
+						"injections:\n" +
+						"  - imageserver: 'https://example.com/imageserver/'\n" +
+						"  - old_imageserver: 'http://kb-images.kb.dk'\n" +
+						"  - origin: 'ds.test'\n";
+		YAML yaml = YAML.parse(new ByteArrayInputStream(yamlStr.getBytes(StandardCharsets.UTF_8)));
+		String solrString = getTransformedFromConfigWithAccessFields(yaml, record);
+		Gson gson = new GsonBuilder().setPrettyPrinting().create();
+		JsonElement je = JsonParser.parseString(solrString);
+		String prettyJsonString = gson.toJson(je);
+		System.out.println(prettyJsonString);
```

Comment:
Same code as `prettyPrintSolrJsonFromPreservica`, right? Time for a helper method :-)